docs/content/SRS/SRS.md

```diff
@@ -163,33 +231,83 @@ _The following Software Requirements Specification for `tinymorph` is using [Vol
 
 ### 10.1 Appearance Requirements
 
-[Insert your content here.]
+LF-A1: The Tinymorph application shall adopt a unified and sophisticated visual design.
+
+Rationale: A uniform and elegant design ensures brand recognition and provides a visually pleasing experience for users, fostering greater engagement and trust in the platform.
+
+LF-A2. The application must utilize a consistent typography system across all user interfaces.
```

Comment:
ig consistent design system in this case (typography is included within a design system)

so does color pallete

Reply:
Included the mentioning of consistent color pallet for the design of our system.